<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>891</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>891</width>
    <height>650</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>TeX Gyre Schola</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>PyBSM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>851</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Linux Biolinum Keyboard O</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PyBoard   and   Serial   communication:    
     Real   time   accelerometer   measure</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Real time measure</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>851</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>85</y>
      <width>851</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Load measure</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>851</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Timer1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Timer2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set threshold auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>95</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set threshold manual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>195</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start ploting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>220</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Wher to save measure:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>59</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes?</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Check if you want to save:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>obrada.npy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>385</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>obrada.npy</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_11">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>385</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>What file:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>385</y>
      <width>94</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_12">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load data</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_13">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>385</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_17">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Z axis measure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>505</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-1.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>505</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-125.255555</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_16">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Y axis measure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>505</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>125.000001</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_15">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>X axis measure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>29</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="geometry">
     <rect>
      <x>1656</x>
      <y>167</y>
      <width>131</width>
      <height>99</height>
     </rect>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAbout">
     <property name="title">
      <string>About</string>
     </property>
     <addaction name="actionAbout_app"/>
     <addaction name="separator"/>
     <addaction name="actionAbout_author"/>
    </widget>
    <addaction name="menuAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_app">
   <property name="text">
    <string>About_app</string>
   </property>
  </action>
  <action name="actionAbout_author">
   <property name="text">
    <string>About_author</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
